docs/testing.qmd

```diff
@@ -0,0 +1,120 @@
+---
+title: Unit testing
+editor:
+  markdown:
+    wrap: sentence
+---
+
+Testing Shiny apps is important to ensure the shiny app functions as expected and to catch any bugs or errors before deployment. It helps maintain code quality, user experience, and prevents potential issues from reaching the end-users.
```

Comment:
```suggestion
Testing is important to ensure your apps continue working as intended. It helps maintain code quality, user experience, and prevents potential issues from reaching the end-users.
```

Reply:
Resubmitted same comment under the new file name. Resolving this comment.